main/src/vocab/nllb_vocab.rs

```diff
@@ -0,0 +1,238 @@
+use std::{
+    collections::{HashMap, HashSet},
+    fs::File,
+    io::BufReader,
+    path::Path,
+};
+
+use serde::Deserialize;
+use snafu::ResultExt;
+
+use crate::error::*;
+
+use super::{
+    base_vocab::{swap_key_values, SpecialTokens},
+    Vocab,
+};
+
+pub const FAIRSEQ_LANGUAGE_CODES: [&str; 202] = [
```

Comment:
Could you please rename (e.g. `EXTENDED_FAIRSEQ_LANGUAGE_CODES`) to avoid confusion with the other language codes?

Reply:
Fixed in #81.